packages/wxt/src/core/zip.ts

```diff
@@ -52,8 +52,12 @@ export async function zip(config?: InlineConfig): Promise<string[]> {
   zipFiles.push(outZipPath);
   await wxt.hooks.callHook('zip:extension:done', wxt, outZipPath);
 
-  // ZIP sources for Firefox
-  if (wxt.config.browser === 'firefox') {
+  // ZIP sources for Firefox or Opera, or when explicitly requested
+  if (
+    wxt.config.browser === 'firefox' ||
+    wxt.config.browser === 'opera' ||
+    config?.zip?.alwaysBuildSourcesZip
```

Comment:
Instead of having this logic here, can we create a `wxt.config.zip.zipSources` boolean that gets set to `true` or `false` inside `resolve-config.ts`? That way this code can just be `if (wxt.config.zip.zipSources) { ... }`.

This is the pattern I've been following with all config - simplify config as much of it as possible inside `resolve-config.ts` so if it's referenced multiple times throughout the code, we don't have to recalculate the value multiple times.

Reply:
Done, idk why but adding / changing something in the config is really hard? You have to manually add it in like 4 places and typecheck doesn't even throw an error if you forget it at one?

Reply:
Hmm, types should throw an error... I guess unless you use optional types, then it won't. I try to use `| undefined` to avoid that.